tests/astro_tests/bossbodenheimer.dat

```diff
@@ -7,13 +7,13 @@
 #-----------------------------
 # Initial conditions variables
 #-----------------------------
-Simulation run id string                    : run_id = BB1
-Run SPH simulation                          : sim = sph
+Simulation run id string                    : run_id = BB1-MFV
+Run SPH simulation                          : sim = mfvmuscl
```

Comment:
Not particularly.  But what we probably should do is something similar to the paper_tests directory and have separate parameters files for each hydrodynamics algorithm we wish to change, or if not that, then at least rename the parameters file to reflect what algorithm is being used (e.g. bb-gradh.dat).  Any thoughts?